### ISSUE TITLE
Official Flatpak

### ISSUE THREAD

**Issue by julianfairfax** [flathub/im.riot.Riot#399]
Is there any chance of this Flatpak becoming official, and thus becoming verified on the Flathub website? Is there anything that I can do to advance this?

**Comment by SISheogorath** [comment]
There was work done upstream to make a flatpak release part of the regular release pipeline, but I don't think that was ever finished: https://github.com/vector-im/element-desktop/pull/293

And since this officially an Element Community build, it's not something that could be marked as official or verified on the Flathub website/store.

At the same time, the same way something official doesn't mean it's good, being unofficial doesn't mean it's bad. So personally, I wouldn't worry to much about the status.

**Comment by t3dium** [comment]
Yeah flatpaks being unverified isn't too much of an issue i'd say - at least regarding the safety of flatpaks. For example this element flatpak only has a couple files on this repo and they're all pretty easy to read, you can also see its fetching from official sources: packages.element.io in the .yaml.

**Comment by AsciiWolf** [comment]
Unverified Flatpaks are not going to be displayed anymore [on Linux Mint](https://www.omgubuntu.co.uk/2024/06/linux-mint-will-hide-unverified-flatpaks) (and probably also other distributions in the future).

**Comment by SISheogorath** [comment]
Well, I guess good for them? I don't see how this will affect us, unless now Element decides they want to step up. The Pull Requests are stale for years now and we are not in the position to make it verified now do I see much benefit in pushing for that.
